docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/pooling/max-pool-14.rst

```diff
@@ -0,0 +1,404 @@
+.. {#openvino_docs_ops_pooling_MaxPool_14}
+
+MaxPool
+=======
+
+
+.. meta::
+  :description: Learn about MaxPool-8 - a pooling operation, which can 
+                be performed on a 3D, 4D or 5D input tensor.
+
+**Versioned name**: *MaxPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Performs the max pooling operation on input.
+
+**Detailed description**: Input shape can be either 3D, 4D, or 5D. The max pooling operation is performed with respect to input shape from the third dimension to the last dimension. If paddings are used, during the pooling calculation their values are ``-inf``. The max pooling operation involves sliding a filter over each channel of a feature map and downsampling by choosing the largest value within the region covered by the filter.
+
+**Attributes**: *Pooling* attributes are specified in the ``data`` node, which is a child of the layer node.
+
+* *strides*
+
+  * **Description**: *strides* is a distance (in pixels) to slide the window on the feature map over the (z, y, x) axes for 3D poolings and (y, x) axes for 2D poolings. For example, *strides* equal to "4,2,1" means sliding the window 4 pixels at a time over depth dimension, 2 over height dimension, and 1 over width dimension.
+  * **Range of values**: integer values starting from 0
+  * **Type**: int[]
+  * **Required**: *yes*
```

Comment:
can we please move this attribute into inputs, and kernel. That is because they are inputs in TF: https://www.tensorflow.org/api_docs/python/tf/raw_ops/MaxPoolV2

Reply:
@p-wysocki Please note that moving `strides` and `kernel_size` from attributes to inputs will affect shape inference and constructor of the MaxPool op.
It means that additional logic in the shape infer and plugins implementation is needed to handle the case when `strides`, `kernel_size` are/aren't Constant.

@rkazants Are there examples of models with non-constant `strides`, `kernel_size`? 
Maybe they are just provided as a Constant or able to be constant folded Tensors?

Reply:
I would be heavily in favor of tracking this change in a different ticket. MaxPool14/AvgPool14 we planned specifically to address ticket 133929 - I have other commitments for the upcoming release which would conflict with introducing all required changes to shape inference and constructors.

If that's alright with you @rkazants let's follow it up in 134084 and get a clear priority on this.

Reply:
I'm voting to divide these changes into separate PRs, to avoid bloaty PR "everywhere everything at once" effect